src/main/java/by/exadel/internship/controller/UserController.java

```diff
@@ -67,4 +67,11 @@ public void saveUserTime(@PathVariable(name = "userId") UUID userId,
                              @RequestBody List<UserTimeSlotDTO> userTimeSlotDTOList) {
         userTimeSlotService.saveUserTime(userTimeSlotDTOList,userId);
     }
+
+    @AdminAccessControl
+    @GetMapping("/getByRole")
```

Comment:
Again.
We repeat it from PR to PR. [Read this article](https://medium.com/@mwaysolutions/10-best-practices-for-better-restful-api-cbe81b06f291). If you confused feel free to ask non clear points. 

```suggestion
    @GetMapping
```